cumulusci/tasks/vlocity/vlocity.py

```diff
@@ -84,3 +96,50 @@ class VlocityDeployTask(VlocitySimpleJobTask):
     """Runs a `vlocity packDeploy` command with a given user and job file"""
 
     command_keyword: Final[str] = "packDeploy"
+
+
+class OmniStudioDeployRemoteSiteSettings(AddRemoteSiteSettings):
+    """Deploys remote site settings needed for OmniStudio.
+    This cannot be configured in cumulusci/cumulusci.yml because
+    the values for the 'url' field are dynamic."""
+
+    task_options: dict = {}
+
+    def _get_options(self) -> RSSOptions:
+        namespace = self.project_config.project__package__namespace
+        if not namespace:
+            raise TaskOptionsError(NO_NAMESPACE_FOUND)
+
+        visualforce_url: str = self.org_config.instance_url.replace(
+            ".my.salesforce.com",
+            f"--{namespace}.{self.org_config.instance_name}.visual.force.com",
+        )
+        legacy_visualforce_url: str = self.org_config.instance_url.replace(
+            ".my.salesforce.com",
+            f"--{namespace}.vf.force.com",
```

Comment:
@jofsky Are we 100% sure this is the project's namespace, not the OmniStudio package's namespace? That seems odd to me.

Reply:
so when I tried vlocity university's trial demo they had the remoteSiteSettings namespaced with vbt_install or something along those lines not omnistudio. @Br4nd0R @davidmreed

Reply:
Here's the latest install instructions: https://docs.vlocity.com/en/Manually-Update-Remote-Site-Settings-for-FlexCards.html

It looks like the domain name uses _the namespace of whatever package in the org contains the FlexCard designer_. Let's make that a user-facing option and not attempt to infer which namespace to use.

Reply:
done